### DISCUSSION TITLE
Support multiple policies for an External User

### DISCUSSION THREAD

**Discussion by shivakumargowda12** [minio/minio#16817] [Q&A] [answered]
Just wanted to confirm, is multiple policy supported for a User logged in through Keycloak, when policy claim has comma separated policy names?
This request had be raised back here - https://github.com/minio/minio/issues/8651

I have understood, this is achievable for local users in minio. - https://github.com/minio/minio/pull/9550
But, how can we do it for an external User ? Am I missing something here ?

**Comment by harshavardhana** [comment, marked as answer]
Your policy claim can be of the form

```
"policy": [
  "readwrite",
  "disableadmin",
]
```

Or 

```
"policy": "readwrite,disableadmin"
```

**Reply by shivakumargowda12** [reply]
Thanks for the quick response. I had tried updating **policy** User attribute in Keycloak with comma separated values (**policy1, policy2**). It dint work.
Minio Server Version - **RELEASE.2023-02-27**

**Reply by shivakumargowda12** [reply]
I am bombarding with many doubts here, sorry for that
1. Once I enable OIDC in minio, it gives only OIDC login option through UI, why not User Credentials or STS ? I have local users also created
2. I want to upload CA cert of my Keycloak Server onto minio, tried specifying **--certs-dir** option. Its not working. It still shows as **Untrusted Certificate Error**, and login page wont load. But with -**-certs-dir** option, SSL is working for minio
3. Also, where can we see debug logs ? **mc admin logs** wont give much information

**Comment by shivakumargowda12** [comment]
Marking this as resolved, as it is working as expected
